scripts/validations.py

```diff
@@ -0,0 +1,110 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results):
+    return (result
+            for result in results
+            if not result['metadata'].get('listed', True))
+
+
+def severe_validations(results):
+    return (result
+            for result in results
+            if result['signing_summary']['high'] > 0)
```

Comment:
We want results which have any signing messages that aren't `trivial`, so, `high + medium + low > 0`. Or, alternatively, you should just be able to check the "passed_auto_validation" property.